src/components/Class/ClassForm.js

```diff
@@ -656,6 +655,23 @@ function ClassForm({
     }
   }, [selectedPartners]);
 
+  const pathwayName = partnerPathwayId?.map((item) => {
+    const foundDictionary = data.Pathways.data.pathways.find(
+      (dict) => dict.id === item
+    );
+    const name = foundDictionary.id ? foundDictionary.name : null;
```

Comment:
`const name = foundDictionary.id ? foundDictionary.name : null;`
Remove this line. You don't need it as you can directly take id and name from the object you're getting.